kickoff/static/kickoff.js

```diff
@@ -33,30 +33,196 @@ function initialSetup() {
 }
 
 function viewReleases() {
-    toLocalDate();
-    // initial sorting by SubmittedAt (descending)
-    // and then saving user table state using localStorage
-    $('#reviewed').dataTable({
-        'bJQueryUI': true,
-        'aaSorting': [[3, 'desc']],
-        'bStateSave': true,
-        'fnStateSave': function(oSettings, oData) {
-        localStorage.setItem('DataTables_reviewed' + window.location.pathname, JSON.stringify(oData));
-    },
-        'fnStateLoad': function(oSettings) {
-        return JSON.parse(localStorage.getItem('DataTables_reviewed' + window.location.pathname));
-    }
+    $('#reviewed').DataTable({
+        'bServerSide': true,
+        'sAjaxSource': '/releases/releaseslist',
+        'sAjaxDataProp': 'releases',
+        'aaSorting': [[4,'desc']],
```

Comment:
Could you replace 4 by a variable to explicit what it is about? (I know it is not your fault)